.travis.yml

```diff
@@ -9,11 +9,11 @@ services:
 
 # Disable automatic submodule fetching ...
 git:
+    depth: 9999999
```

Comment:
Actually since the project has a `src` directory, `SRC=.` is not even necessary :)

Then of course you need to add the package description files in `pkg` as usual.